config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -50,7 +50,7 @@ presubmits:
           - name: GINKGO_FOCUS
             value: "Workload cluster creation"
           - name: GINKGO_SKIP
-            value: "Creating a GPU-enabled cluster"
+            value: "Creating a GPU-enabled cluster|Creating a Windows enabled VMSS cluster"
```

Comment:
There are actually two different contexts for Windows right now https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1036/files#diff-93ce65d81f5c1510135f6035d43f678bf9b5962b8e4ee45105e3345a1852efc6R354

How can we make this more generic so it doesn't break in the future if more windows tests are added?

Reply:
Could we use a well known set of test of tags?  Something like `[presubmitt optiontal]` or more specific like `[Windows]`? Are there any like this in use in Cluster API anywhere?

Reply:
we can make a regex that filters the `Windows` keyword

or we create another `Describe` and add all windows tests there

Reply:
Using `Windows` works for me

Reply:
updated PTAL
@CecileRobertMichon @jsturtevant